.gitignore

```diff
@@ -6,6 +6,7 @@ logs
 *.log.*
 npm-debug.log*
 .audit.json
+*-audit.json
```

Comment:
huh?

Reply:
it appears npm audit log filename patterns changed with some version of npm

Reply:
oh lovely